<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 104, 113);
background-color: rgb(147, 155, 184);</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <widget class="QListWidget" name="evidencesListWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>581</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(247, 247, 247);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="savePushButton">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>230</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>410</y>
          <width>411</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="idLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="measurementLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="clarityLevelLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="quantityLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="photographLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>410</y>
          <width>171</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Lucida Sans Typewriter</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Lucida Sans Typewriter</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Measurement:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Lucida Sans Typewriter</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Image clarity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Lucida Sans Typewriter</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Quantity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Lucida Sans Typewriter</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Photograph:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>620</y>
          <width>581</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addPushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);
border-color: rgb(85, 255, 255);</string>
           </property>
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updatePushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string>UPDATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="listpushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string>LIST</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>670</y>
          <width>221</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="UndoPushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string>UNDO</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redoPushButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(239, 239, 239);</string>
           </property>
           <property name="text">
            <string>REDO</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Y</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QCheckBox" name="seeUsers_checkBox">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>670</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans Typewriter</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OPEN CASE FILES</string>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>411</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans Typewriter</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>List of collected evidences</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Standard</string>
       </attribute>
       <widget class="QListWidget" name="savedListWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>481</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="filterPushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>590</y>
          <width>81</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string>FILTER</string>
        </property>
       </widget>
       <widget class="QListWidget" name="filterListWidget">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>560</y>
          <width>311</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>590</y>
          <width>291</width>
          <height>59</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Lucida Sans Typewriter</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Measurement:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Lucida Sans Typewriter</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Quantity:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="measurementFilterLineEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="quantityFilterLineEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="nextListWidget">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>320</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nextPushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>330</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mylistOpenpushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>490</y>
          <width>481</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string>MYLIST</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openPushButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>440</y>
          <width>481</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 239, 239);</string>
        </property>
        <property name="text">
         <string>OPEN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>411</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans Typewriter</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>My list of gathered evidences</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="pie_checkBox">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>710</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="pie_chart_check_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>390</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PieChart</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
